smart_campus/app/templates/app/add_station.html

```diff
@@ -0,0 +1,75 @@
+{% extends "app/base.html" %}
+{% load static %}
+
+{% block title %} Smart Campus{% endblock %}
+
+{% block stylesheet %}{% static "app/css/add_station.css" %}{% endblock %}
+
+{% block content %}
+<div class="container">
+	<form class="" action="" method="POST">            
+    {% csrf_token %}
```

Comment:
One more indentation on this line.
Also, the indentation of this HTML file should be 2 spaces.